rllib/utils/exploration/epsilon_greedy.py

```diff
@@ -153,33 +148,5 @@ def _get_torch_exploration_action(self, q_values, explore, timestep):
 
     @override(Exploration)
     def get_info(self):
-        """Returns the current epsilon value.
-
-        Returns:
-            Union[float,tf.Tensor[float]]: The current epsilon value.
-        """
-        return self.epsilon_schedule(self.last_timestep)
-
-    @override(Exploration)
```

Comment:
Removed this unimplemented state code for now. We can add it back as needed later.